portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+     <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+
+  </head>
+  <body>
+
+      <h1 class="jumbotron"> <center> Gallery </center> </h1>
+<nav>
+      <ul>
+   <li><a href="index.html">Home</a></li>
+  <li><a href="about.html">About</a></li>
+  <li><a href="gallery.html">Gallery</a></li>
+  <li><a href="poems.html"> Favorite Poems</a></li>
+  <li><a href="topten.html"> My Top Ten Anime </a></li>
+      </ul>
+</nav>
+      
+      
+          
+       <div class="container">
+      <img src="media/portfolioimage.jpg" alt="Eric on a bridge." class="img-circle" style="width:410px;height:410px;">
+      <h3>20th</h3>
+       </div>
+       
+
+        
+          <div class="container">
+      <img src="media/firstgpic.jpg" alt="MTV EP 2019" class="img-circle" style="width:410px;height:410px;">
+       <h3>MTV Office 1</h3>
+       </div>
+        
+
+        
+          <div class="container">
+      <img src="media/secondgpic.jpg" alt="MTV EP 2019" class="img-circle" style="width:410px;height:410px;">
+       <h3>MTV Office 2</h3>
+       </div>
+        
+
+        
+          <div class="container">
+      <img src="media/moonlight.jpg" alt="Expensive mistake." class="img-circle" style="width:410px;height:410px;">
+       <h3>Expensive Mistake</h3>
+       </div>
+        
+
+        
+          <div class="container">
+      <img src="media/elections.jpg" alt="Dapper days" class="img-circle" style="width:410px;height:410px;">
+       <h3>Dapper Days</h3>
+       </div>
+        
+
+
+
+      
+
+    <!-- Javascript for Bootstrap -->
+  <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js" integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
```

Comment:
Same here with common imports, you can create a script.js file and just add these to that file and each of your html files can just import script.js